docs/install/kubernetes.md

```diff
@@ -234,53 +247,43 @@ artifact. You can use template values in the artifact path. The supported values
 
 Configurable values are documented in the `values.yaml`, and the `values.yaml` of all sub charts. Override those [in the normal methods](https://helm.sh/docs/chart_template_guide/values_files/).
 
-## Uninstalling the chart
-
+## Uninstalling the Chart
 ```bash
-helm --namespace mlrun uninstall mlrun-kit
+helm --namespace mlrun uninstall mlrun-ce
 ```
 
-```{admonition} Terminating pods and hanging resources
-This chart generates several persistent volume claims and also provisions an NFS
-provisioning server, giving persistency (via PVC) out of the box.
-Because of the persistency of PV/PVC resources, after installing this chart, PVs and PVCs are created.
-Upon uninstallation, any hanging / terminating pods hold the PVs and PVCs respectively, since those
-prevent their safe removal.
+#### Note on terminating pods and hanging resources
+It is important to note that this chart generates several persistent volume claims in order to provide the user 
```

Comment:
This chart generates several persistent volume claims that provide persistency (via PVC) out of the box.